pkg/frontend/querymiddleware/stats_test.go

```diff
@@ -62,47 +62,16 @@ func Test_queryStatsMiddleware_Do(t *testing.T) {
 				Step:         step,
 			},
 		},
-		{
```

Comment:
no longer a case as the query will have already been validated and parsed before it is in the request object